### PR TITLE
use service instead of command

### PR DESCRIPTION
**Pull request by benrudolph**
@millerdev fixes the issue about starting riak and it failing hard. riak is now running on staging. you should also check out `ansible/roles/riakcs/README.md`

cc: @emord

### CODE REVIEW

File: ansible/roles/riakcs/tasks/admin_user.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 # Note this can't be notify because we need it to restart immediately
 - name: Restart RiakCS
-  command: riak-cs restart
+  service: name=riak-cs state=started enabled=yes
```
Comment (millerdev) [inline ansible/roles/riakcs/tasks/admin_user.yml]: Why are there two tasks in the file called `Restart RiakCS`? (see also https://github.com/dimagi/commcarehq-ansible/blob/fix-riak-start/ansible/roles/riakcs/tasks/admin_user.yml#L67)

Comment (millerdev) [inline ansible/roles/riakcs/tasks/admin_user.yml]: Also this is making sure the service is started. It does not restart the service if it's already running ([docs](http://docs.ansible.com/ansible/service_module.html#options)). Use `state=restarted` if it should always restart the service.

Comment (benrudolph) [inline ansible/roles/riakcs/tasks/admin_user.yml]: you're right, this should just be named Start RiakCS. we just need to make sure it's running. the command below needs to be a restart for the changes to take effect. will update that

Comment (benrudolph) [inline ansible/roles/riakcs/tasks/admin_user.yml]: the reason there's a bunch of these is that you can run this playbook in portions using tags/defaults and i make no assumption up the status of the service (beyond the fact that it's installed)

Comment (benrudolph) [inline ansible/roles/riakcs/tasks/admin_user.yml]: (actually it is supposed to be restart)

Comment (benrudolph) [inline ansible/roles/riakcs/tasks/admin_user.yml]: https://github.com/dimagi/commcarehq-ansible/commit/3b98546be6633d23fe65ea5eed05bcb1e8192d5e

**Pull request merged**